product_gtin/product_gtin.py

```diff
@@ -37,6 +37,53 @@ def check_ean(eancode):
         int(eancode)
     except:
         return False
+    check = True
+    if len(eancode) == 8:
+        check = check_ean8(eancode)
+    if len(eancode) == 11:
+        check = check_ean11(eancode)
+    if len(eancode) == 12:
+        check = check_upc(eancode)
+    if len(eancode) == 13:
+        check = check_ean13(eancode)
+    if len(eancode) == 14:
+        check = check_gtin14(eancode)
+    return check
```

Comment:
Hello, I think at bare minimum you should use elif instead of just if. Or eventually some kind of dynamic dispatch such as http://stackoverflow.com/questions/11479816/what-is-the-python-equivalent-for-a-case-switch-statement as there is no case switch in Python.